core/components/atoms/chip/__stories__/ConfigurableLabels.story.jsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
```

Comment:
Stories should look something like the follow: 
<img width="1007" alt="Screenshot 2022-12-07 at 6 53 35 PM" src="https://user-images.githubusercontent.com/3583587/206190821-c2930c13-6cd2-4ca5-806a-acc106d19646.png">